simuleval/data/dataloader/dataloader.py

```diff
@@ -52,15 +56,28 @@ def get_source(self, index: int) -> Any:
     def get_target(self, index: int) -> Any:
         return self.preprocess_target(self.target_list[index])
 
+    def get_tgt_lang(self, index: int) -> Any:
+        if self.tgt_lang_list is not None and index < len(self.tgt_lang_list):
+            return self.preprocess_tgt_lang(self.tgt_lang_list[index])
+        else:
+            return None
+
     def __getitem__(self, index: int) -> Dict[str, Any]:
-        return {"source": self.get_source(index), "target": self.get_target(index)}
+        return {
+            "source": self.get_source(index),
+            "target": self.get_target(index),
+            "tgt_lang": self.get_tgt_lang(index),
+        }
 
     def preprocess_source(self, source: Any) -> Any:
         raise NotImplementedError
 
     def preprocess_target(self, target: Any) -> Any:
         raise NotImplementedError
 
+    def preprocess_tgt_lang(self, tgt_lang: Any) -> Any:
+        raise NotImplementedError
+
```

Comment:
```suggestion
```

Also, even if we want a a preprocessor for tgt_lang, throwing a `NotImplementedError` here would break some of the dataloaders (not visible to you) that derives from this base class and don't have this new method implemented.